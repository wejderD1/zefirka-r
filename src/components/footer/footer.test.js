import { render, screen } from "@testing-library/react";
import Footer from "./footer";

describe("Footer component", () => {
  it("Footer renders", () => {
    render(<Footer />);
  
    expect(screen.getByRole("contentinfo")).toBeInTheDocument();
    expect(screen.getByText(/Yurii/i)).toBeInTheDocument();
  });

  it("Footer snapshot", () => {
    const view = render(<Footer />);

    expect(view).toMatchSnapshot();
  })
})
