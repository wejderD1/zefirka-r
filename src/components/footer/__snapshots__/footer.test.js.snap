// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        id="footer"
      >
        <div
          class="container"
        >
          <div
            class="footer-inner"
          >
            <h2
              class="subtitle footer-title"
            >
              © 
              2024
              , Yurii Hnatiuk, wejderd1@gmail.com
            </h2>
            <div
              class="social-container"
            >
              <a
                class="social-links"
                href="https://www.facebook.com/groups/748009266201827/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  class="fab fa-facebook-f"
                />
              </a>
              <a
                class="social-links"
                href="#"
              >
                <i
                  class="fab fa-instagram"
                />
              </a>
              <a
                class="social-links"
                href="#"
              >
                <i
                  class="fab fa-tiktok"
                />
              </a>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      id="footer"
    >
      <div
        class="container"
      >
        <div
          class="footer-inner"
        >
          <h2
            class="subtitle footer-title"
          >
            © 
            2024
            , Yurii Hnatiuk, wejderd1@gmail.com
          </h2>
          <div
            class="social-container"
          >
            <a
              class="social-links"
              href="https://www.facebook.com/groups/748009266201827/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-facebook-f"
              />
            </a>
            <a
              class="social-links"
              href="#"
            >
              <i
                class="fab fa-instagram"
              />
            </a>
            <a
              class="social-links"
              href="#"
            >
              <i
                class="fab fa-tiktok"
              />
            </a>
          </div>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
